src/System.Runtime.Extensions/src/System/Diagnostics/Stopwatch.cs

```diff
@@ -4,10 +4,10 @@
 
 namespace System.Diagnostics
 {
-    // This class uses high-resolution performance counter if installed hardware 
-    // does not support it. Otherwise, the class will fall back to DateTime class
-    // and uses ticks as a measurement.
-
+    // This class uses high-resolution performance counter if the installed
+    // hardware supports it. Otherwise, the class will fall back to DateTime
+    // class and uses ticks as a measurement.
```

Comment:
> DateTime class

DateTime isn't a class. I would just write "DateTime" and leave struct out of this.

Reply:
Sure.